src/NuGet.Configuration/PackageSource/PackageSourceProvider.cs

```diff
@@ -29,22 +30,24 @@ public PackageSourceProvider(ISettings settings)
         /// <param name="settings">Specifies the settings file to use to read package sources.</param>
         /// <param name="providerDefaultSources">Specifies the default sources to be used as per the PackageSourceProvider. These are always loaded
         /// Default Feeds from PackageSourceProvider are generally the feeds from the NuGet Client like the NuGetOfficialFeed from the Visual Studio client for NuGet</param>
-        public PackageSourceProvider(ISettings settings, IEnumerable<PackageSource> providerDefaultSources)
-            : this(settings, providerDefaultSources, migratePackageSources: null)
+        public PackageSourceProvider(ISettings settings, IEnumerable<PackageSource> providerDefaultPrimarySources, IEnumerable<PackageSource> providerDefaultSecondarySources)
```

Comment:
The XML comments don't explain what "secondary" is.  Please update those to explain; I think there could be a word better than "secondary" too--perhaps "legacy".